.bowerrc

```diff
@@ -0,0 +1,3 @@
+{
+    "directory": "static/libs/bower_components"
```

Comment:
It's more consistent manage dependencies with package manager instead of relying on files magically put into repo without version kept.